apps/hash-ai-worker-ts/src/activities/graph.ts

```diff
@@ -0,0 +1,50 @@
+import type {
+  EntityEmbedding,
+  EntityTypeStructuralQuery,
+  GraphApi,
+} from "@local/hash-graph-client";
+import type {
+  AccountId,
+  EntityTypeRootType,
+  PropertyTypeWithMetadata,
+  Subgraph,
+} from "@local/hash-subgraph";
+import { getPropertyTypes } from "@local/hash-subgraph/stdlib";
+
+export const createGraphActivities = ({
+  graphApiClient,
+}: {
+  graphApiClient: GraphApi;
+}) => ({
+  async getEntityTypesByQuery(params: {
+    authentication: {
+      actorId: AccountId;
+    };
+    query: EntityTypeStructuralQuery;
+  }): Promise<Subgraph<EntityTypeRootType>> {
+    return graphApiClient
+      .getEntityTypesByQuery(params.authentication.actorId, params.query)
+      .then((response) => response.data as Subgraph<EntityTypeRootType>);
```

Comment:
A bit safer if the return type of `getEntityTypesByQuery` ever changes
```suggestion
      .then((response) => mapGraphApiSubgraphToSubgraph(response.data));
```

above
```ts
import {
  getPropertyTypes,
  mapGraphApiSubgraphToSubgraph,
} from "@local/hash-subgraph/stdlib";
```